### PR TITLE
Add ability to edit users by admin

### PR DESCRIPTION
**Pull request by freevova**

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Please review.

File: app/views/users/edit.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %h1.title= t('edit.edit_profile')
-- if current_user.admin?
+- if current_user.admin? && @edit_admin
```
Comment (EnragedWildkin) [inline app/views/users/edit.html.haml]: This will raise exception for unlogged users.

File: app/assets/javascripts/contracts.coffee [hunk 2]
```diff
@@ -16,3 +17,14 @@ $ ->
     else
       $('.contract_end_date').show('slow')
       $('#contract_end_date').attr('required', true)
+
+# Edit user chosen in the select tag(#contract_user_id)
+class window.AdminEditUser
```
Comment (EnragedWildkin) [inline app/assets/javascripts/contracts.coffee]: Can user have missing fields at the moment? Do we have check for missing fields before generating contract? 
If not, maybe it's better for now to edit user only from users index page.

**Comment by EnragedWildkin** [comment]
@freevova Please check my comments.

File: app/assets/javascripts/contracts.coffee (hunk 2 above)
Comment (freevova) [inline app/assets/javascripts/contracts.coffee]: Add posibility to edit users from users page.

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Pull request closed without merging**